state/futures/reducer.ts

```diff
@@ -171,7 +171,7 @@ export const FUTURES_INITIAL_STATE: FuturesState = {
 		leverageInput: '0',
 	},
 	tradePanelDrawerOpen: false,
-	marketFundingRates: [],
+	marketFundingRates: {} as FuturesState['marketFundingRates'],
```

Comment:
You shouldn't need to do this as the whole object state is defined as `FuturesState`

Reply:
Oh yea I think I usually wrap the object type with Partial to deal with this `Partial<Record<...>>`

Reply:
Switched to that.